src/Compiler/FSComp.txt

```diff
@@ -1037,7 +1037,8 @@ lexUnexpectedChar,"Unexpected character '%s'"
 1154,lexOutsideDecimal,"This number is outside the allowable range for decimal literals"
 1155,lexOutsideThirtyTwoBitFloat,"This number is outside the allowable range for 32-bit floats"
 1156,lexInvalidNumericLiteral,"This is not a valid numeric literal. Valid numeric literals include 1, 0x1, 0o1, 0b1, 1l (int/int32), 1u (uint/uint32), 1L (int64), 1UL (uint64), 1s (int16), 1us (uint16), 1y (int8/sbyte), 1uy (uint8/byte), 1.0 (float/double), 1.0f (float32/single), 1.0m (decimal), 1I (bigint)."
-1157,lexInvalidByteLiteral,"This is not a valid byte literal"
+1157,lexInvalidAsciiByteLiteral,"This is not a valid ASCII byte literal. Value must be < 128y."
+1157,lexInvalidTrigraphAsciiByteLiteral,"This is not a valid ASCII byte literal. Value should be < 128y.\nNote: In a future F# version this Warning will be promoted to Error!"
```

Comment:
Not sure we can commit to this right now. We better remove the note.

Reply:
Same applies below.

Reply:
It's good to write it that way actually, it should be an error eventually. I think this is appropriate for warnings which have an easy resolution.
At the same time,"future"  is not binding -> there is no harm if this takes 3,5.. or more versions to happen.

Reply:
Alright, yeah, fair enough, this is not binding.